index.js

```diff
@@ -41,6 +42,15 @@ for (const file of commandFiles) {
     }
   }
 }
+for (const file of bankCommandFiles) {
```

Comment:
we should rewrite this to handle commands/name/* or something... but this is fine for now (works differently for admin code so something to think about)